cmd/juju/commands/bootstrap.go

```diff
@@ -534,6 +534,7 @@ func (c *bootstrapCommand) Run(ctx *cmd.Context) (resultErr error) {
 		AdminSecret:    config.bootstrap.AdminSecret,
 	}
 	bootstrapParams := bootstrap.BootstrapParams{
+		ControllerName:            c.controllerName,
```

Comment:
need to enforce controller name as a mandatory arg for k8s bootstrap

Reply:
existing logic is to generate a controllerName in `c.defaultControllerName(cloud)` if the `c.controllerName` was not provided by user.
So it's never empty.